src/ressources/customers/ReviewsNbField.tsx

```diff
@@ -0,0 +1,31 @@
+import { Typography, Stack } from "@mui/material";
+import CommentIcon from "@mui/icons-material/Comment";
+import {
+  useGetList,
+  useRecordContext,
+  UseRecordContextParams,
+} from "react-admin";
+import { Customer } from "./Customers";
+
+export const ReviewsNbField = (props: UseRecordContextParams<Customer>) => {
+  const record = useRecordContext<Customer>(props);
+  const { data, total, isLoading, error } = useGetList("reviews", {
+    pagination: { page: 1, perPage: 5 },
+    filter: { customer_id: record.id },
+  });
```

Comment:
Don't need `data`.
If you're only interested in the `total`, you may as well request only `perPage: 1` item to optimize.